layers.go

```diff
@@ -1156,23 +1153,24 @@ func (s *store) newLayerStore(rundir, layerdir, imagedir string, driver drivers.
 }
 
 func newROLayerStore(rundir string, layerdir string, driver drivers.Driver) (roLayerStore, error) {
-	lockfile, err := lockfile.GetROLockFile(filepath.Join(layerdir, "layers.lock"))
+	roLockfile, err := lockfile.GetROLockFile(filepath.Join(layerdir, "layers.lock"))
 	if err != nil {
 		return nil, err
 	}
+	// Ignore errors if the lock file could not be opened
+	mountsLockfile, _ := lockfile.GetLockFile(filepath.Join(rundir, "mountpoints.lock"))
 	rlstore := layerStore{
-		lockfile:       newMultipleLockFile(lockfile),
-		mountsLockfile: nil,
+		lockfile:       newMultipleLockFile(roLockfile),
+		mountsLockfile: mountsLockfile,
 		rundir:         rundir,
 		jsonPath: [numLayerLocationIndex]string{
 			filepath.Join(layerdir, "layers.json"),
 			filepath.Join(layerdir, "volatile-layers.json"),
 		},
 		layerdir: layerdir,
-
-		byid:    make(map[string]*Layer),
-		byname:  make(map[string]*Layer),
-		bymount: make(map[string]*Layer),
+		byid:     make(map[string]*Layer),
+		byname:   make(map[string]*Layer),
+		bymount:  make(map[string]*Layer),
 
 		driver: driver,
 	}
```

Comment:
Also add the `rlstore.mountsLastWrite is initialized inside rlstore.load().` comment below, matching the other constructor.